src/index.html

```diff
@@ -1,22 +1,36 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
-    <title>Moyo header</title>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Moyo Header</title>
     <link
-      rel="stylesheet"
-      href="./style.css"
-    />
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"/>
+    <link rel="stylesheet" href="style.css"/>
```

Comment:
Keep your attributes correctly formatted. If the HTML-element has a long attribute value or a number of attributes is more than 2 - start each one, including the first, on a new line with 2-space indentation related to the tag.